### PR TITLE
Porfolio mobile website work section PR

### PR DESCRIPTION
**Pull request by codecaiine**
- I added HTML and CSS to work_section branch
- I used Flexbox to my code
- I used Grid
- I fixed linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,58 +13,251 @@
...
-        <div class="collapse navbar-collapse justify-content-between" id="navbarCollapse">
-            <div class="navbar-nav"> </div>
-            <div class="navbar-nav">
-                <a href="#" class="nav-item nav-link active">Porfolio</a>
-                <a href="#" class="nav-item nav-link">About</a>
-                <a href="#" class="nav-item nav-link">Contact</a>
-            </div>
-        </div>
-        </div>
+    <nav class="navbar">
```
Comment (arslanbisharat) [inline index.html:16]: - [ ] this section  should be enclosed in section tag

File: index.html [hunk 2]
```diff
@@ -16,58 +13,251 @@
...
+                         <a href="#"> About </a>
+                      </li>
+                      <li class="list-item">
+                         <a href="#"> Contact </a>
+                      </li>
+                 </ul>
+             <div class="menu">
+                  <div class="menu-line"></div>
+                  <div class="menu-line"></div>
+                  <div class="menu-line"></div>
```
Comment (arslanbisharat) [inline index.html:35]: - [ ] thes should be enclosed it on nav section

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested  

Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 1 above)
Comment (codecaiine) [inline index.html]: Hi ,
Can you tell me what's wrong with this? I don't really understand.  In screenshot, you will see what I did. 

Screenshots

File: index.html (hunk 2 above)
Comment (codecaiine) [reply index.html]: I think, this is already in nav section or I don't really understand what you m

Screenshots

**Review by arslanbisharat — COMMENTED** [review]
Hi there, 
I am sorry I made a mistake while reviewing the project. I thought the menu should be in nav tag and you missed it but that's not true, you implemented that. 

Sorry for the trouble. Please request another review.

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html]: Well resolved

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Yannick, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 💯 👍.
Congratulations! 🎉 🎉 🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**